tests/unset.bat

```diff
@@ -0,0 +1,8 @@
+cd %1
```

Comment:
What is this script for? I don't see anything calling it?

Reply:
The `setup.bat` script in the same folder creates some read-only, no-access files.

This `unset.bat` script can help you restore permissions and delete the test data folder, when you want to rebuild (`meson build -C build --wipe`) or delete the entire build folder.

This script is not called anywhere, it is just a development tool